bentoml/_internal/models/model.py

```diff
@@ -42,8 +42,8 @@
     from ..types import PathType
 
     class ModelSignatureDict(t.TypedDict, total=False):
-        batch_dim: tuple[int, int]
-        batchable: bool
+        batch_dim: tuple[int, int] | int | None
+        batchable: bool | None
         input_spec: tuple[AnyType] | AnyType | None
         output_spec: AnyType | None
```

Comment:
Does `total=False` not do the optional bit by default?

Reply:
TIL